docs/metrics/customizing-the-sdk/README.md

```diff
@@ -372,6 +372,9 @@ OpenTelemetry SDK comes with the following `ExemplarFilters` (defined on
 The `SetExemplarFilter` extension method on `MeterProviderBuilder` can be used
 to set the desired `ExemplarFilterType` and enable `Exemplar` collection:
 
+> [!NOTE]
+> The `SetExemplarFilter` API was added in the `1.9.0` release.
```

Comment:
can we document the default as well? i.e make it explicit that `AlwaysOff` is the default.

Reply:
something like:
The default ExemplarFilter is AlwaysOff i.e Exemplars feature is off by default.
You can enable it by using SetExemplarFilter, which will enable Exemplars for **all type** of metrics.

There is an experimental ENV variable, which allow you to enable Exemplars for Histogram types only, given exemplars are the most valuable for histograms.

OTEL_DOTNET_EXPERIMENTAL_METRICS_EXEMPLAR_FILTER_HISTOGRAMS=trace-based will enable Exemplars for Histograms only.

Note that there is some cpu/memory/network overhead to compute/store/export Exemplars, which is why, Exemplars are disabled by default.

Reply:
And mention that - there is ability to turn on/off Exemplar for each individual metric today. (if that is allowed via experimental feature, we can mention that here.)

Reply:
I pushed some changes: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5611/commits/5aafc2f2c006a35d759d2942a2bc35db396f1dcb

LMK

> And mention that - there is ability to turn on/off Exemplar for each individual metric today. (if that is allowed via experimental feature, we can mention that here.)

I'm going to do a different PR which adds docs for the experimental `ExemplarReservoir` stuff and shows how to use the view api to set a custom reservoir.